packages/trader/src/Modules/Trading/Components/Form/Purchase/__tests__/contract-info.spec.tsx

```diff
@@ -37,6 +37,7 @@ const default_mock_props: React.ComponentProps<typeof ContractInfo> = {
         profit: '',
         returns: '',
         spot: 0,
+        validation_params: undefined,
```

Comment:
perhaps it would be better if we didn't send an `undefined` field to BE but rather omit it. what do you think?)

Reply:
Unfortunately, it's a TS error, which will gone after deriv/api-types. I'll try to update